### PR TITLE
Remove redundant argument and improve error message

### PR DESCRIPTION
**Pull request by findepi**
## Which issue does this PR close?

None

## Rationale for this change

Simplify the code

## What changes are included in this PR?

Remove redundant argument and improve error message

## Are these changes tested?

No

## Are there any user-facing changes?

No

### CODE REVIEW

**Comment by findepi** [comment]
@comphead would you want to take a look? thanks!

File: datafusion/functions/src/string/common.rs [hunk 1]
```diff
@@ -129,14 +128,13 @@ fn string_view_trim<'a, T: OffsetSizeTrait>(
         }
         other => {
             exec_err!(
-            "{trim_type} was called with {other} arguments. It requires at least 1 and at most 2."
+            "trim was called with {other} arguments. It requires at least 1 and at most 2."
```
Comment (comphead) [inline datafusion/functions/src/string/common.rs:131]:
```suggestion
            "Function TRIM was called with {other} arguments. It requires at least 1 and at most 2."
```

**Review by comphead — APPROVED** [review]
Thanks @findepi its lgtm please add the PR description

**Comment by findepi** [comment]
thanks, updated!

**Pull request merged**